src/unit/categories.js

```diff
@@ -0,0 +1,11 @@
+define(function() {
+
+/**
+ * categories()
+ *
+ * Return all unit categories.
+ */
+return [ "acceleration", "angle", "area", "duration", "length", "mass", "power", "pressure",
+"speed", "temperature", "volume" ];
```

Comment:
An alternative to this approach is generating this dynamically once and keep it cached for further uses. We could keep it as is and improve later though.